selfdrive/controls/lib/drive_helpers.py

```diff
@@ -1,5 +1,5 @@
 import numpy as np
-from opendbc.car.vehicle_model import ACCELERATION_DUE_TO_GRAVITY
+from opendbc.car import ACCELERATION_DUE_TO_GRAVITY
```

Comment:
note for myself for later: probably doesn't make sense to import these from opendbc. it's fine to duplicate these fundamental constants across projects and prevents issues like this

cc @sshane

Reply:
decided to just do it here: https://github.com/commaai/openpilot/pull/35866/commits/8b1d3bdcbcc2498e660ce1b0febb6de261c7a5b3

Reply:
yeah, specially because the small delta caused failing pipelines and I bet they could cause safety blocks on panda given just the right circumstances